relion/tests/test_convert_relion.py

```diff
@@ -781,23 +757,30 @@ def setUpClass(cls):
         setupTestOutput(cls)
         cls.ds = DataSet.getDataSet('relion31_tutorial_precalculated')
 
+    def __readParticles(self, partsStar, outputSqlite=None, **kwargs):
+        outputSqlite = outputSqlite or self.getOutputPath('particles.sqlite')
+        print("<<< Reading star file: \n   %s\n" % partsStar)
+        cleanPath(outputSqlite)
+        print(">>> Writing to particles db: \n   %s\n" % outputSqlite)
+        partsSet = SetOfParticles(filename=outputSqlite)
+        convert.readSetOfParticles(partsStar, partsSet, **kwargs)
+        return partsSet
+
     def test_readSetOfParticles(self):
+        return
```

Comment:
@delarosatrevin the code below is not reachable, is this intended?

Reply:
Fixed.